RunForrest/src/css/body-content.scss

```diff
@@ -0,0 +1,164 @@
+@import "_colors.scss";
+@import "_variables.scss";
+
+.frame{
+    height: 304px;
+    background-image: url("../images/banner-mobile.png");
+    background-position: center;
+    background-size: cover;
+}
+
+hr{
+    background-color: $grey60;
+}
+.main-title{
+    font-size: 24px;
+    line-height: 28px;
+    margin: 15px 95px 15px 27px;
+    width: 70%;
+}
+.search-bar{
+    @extend %flexRow;
+    align-items: center;
+    border: 2px solid;
+    border-radius: 20px;
+    border-color: $grey;
+    height: 44px;
+    margin: 15px 30px 17px 27px;
+
+    &__title{
+        font-size: 18px;
+        font-weight: normal;
+        width: 250px;
+        color: $grey;
+        margin: 13px 0px 10px 15px;
+    }
+
+    &__icon{
+        width: 23px;
+        height: 23px;
+        background-image: url("../images/lupa.png");
+        background-position: center;
+        background-size: cover;
+        margin: 8px 20px 13px 0px;
+    }
+}
+
+
+.box-agents-title{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: space-between;
+    align-items: center;
+
+    &__name{
+        font-size: 18px;
+    }
+    &__icon{
+        width: 26px;
+        height: 26px;
+        background-image: url("../images/certified.png");
+        background-position: center;
+        background-size: cover;
+    }
+
+}
+
+#agents{
+    @extend %flexRow;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+}
+.box-agent{
+    margin-top: 18px;
+    margin-bottom: 25px; 
+    width: 300px;
+}
+.box-agents-content{
+    @extend %flexRow;
+    align-items: center;
+
+    &__photo{
+        height: 120px;
+        background-position: center;
+        background-size: cover;
+    }
+    &__description{
+        text-align: justify;
+        font-size: 12px;
+        font-weight: normal;
+        margin-left: 25px;
+
+    }
+
+}
+
+.box-agents-settings{
+    @extend %flexRow;
+    align-items: center;
+    justify-content: space-between;
+
+    &__button{
+        background-color: $black;
+        color: $white;
+        font-size: 14px;
+        font-weight: normal;
+        border-radius: 18px; 
+        padding: 8px 30px 9px 30px;
+        margin-right: 24px;
+        margin-top: 20px;
+        width: 80px;
+        text-align: center;
+    }
+    &__price{
+        margin-right: 13px;
+        margin-top: 18px;
+        text-align: justify;
+        font-size: 14px;
+    }
+
+}
+
+
```

Comment:
Remove this empty lines. Leave only one, or be consistent between spaces in the whole file (the next @media doesn't have any).